alf/metrics/metrics_test.py

```diff
@@ -214,6 +212,9 @@ def test_average_per_step(self):
                     },
                     'success': to_tensor([1.0, 0.0])
                 }))
+
+        metric = AverageDrivingMetric(example_time_step=trajectories[0])
```

Comment:
Should delete the metric in the beginning of this function?

Reply:
Thought I removed it. Sorry.